tests/python/unittest/test_numpy_ndarray.py

```diff
@@ -614,6 +614,25 @@ def convert(num):
         range(4),
         range(3, 0, -1),
         (range(4,), [1]),
+        # Test Ellipsis ('...')
+        ((1, Ellipsis, -1), False),
```

Comment:
Only the first item of the tuple is needed in this test. Please delete all `False`s.

Reply:
Fixed, thanks!